### ISSUE TITLE
Black logo

### ISSUE THREAD

**Issue by vanillajonathan** [PRQL/prql-brand#7]
In addition to the existing colored logo create a symbolic variant that have no color but black.

**Comment by max-sixty** [comment]
Same as before on this! I would not put any other logos out there unless there's a compelling reason — more risk of confusion than the upside...

**Comment by vanillajonathan** [comment]
But is common that logos are available both in color and in black/white with the black/white being offered as it suitable for print.

**Comment by max-sixty** [comment]
I just don't see any demand here, and so I'm hesitant to publish something we have some responsibility to maintain.

We also don't have the resources to review whether we're doing the correct thing for these. If it's trivial (e.g. just do B&W), then folks can do it themselves. If it's not, then I'm not sure how to know whether we're doing it correctly.

So basically I would vote to not do any more logos unless someone specifically asks to use them and we think it's worthwhile.

Is that reasonable @vanillajonathan ? 

@aljazerzen & @snth am I being reasonable here or overly restrictive?

**Comment by aljazerzen** [comment]
I think that there is no harm in storing logo adaptations to some "requirement" (black & white / Gnome recommendations / Apple recommendations). As @vanillajonathan argues, it helps people who are publishing any material about PRQL.

This does not mean that we are committing to maintaining it. If/when we update the original, all these adaptations will be obsolete and probably removed.

Also, we (as the project) may have objections to some adaptations on the grounds of design.

**Comment by max-sixty** [comment]
If it were free to add excellent versions, I agree it would be marginally positive.

But who is going to do the reviews? How are we going to assess if the colors are good? How do we decide what versions to include?

I'm reminded of https://github.com/PRQL/prql/pull/2147 — this wouldn't be blocking like those, but I'm still wary of taking on any overhead.

I also think we owe it to generous contributors like @vanillajonathan to make productive and impactful use of their time, and this doesn't seem impactful. (Though I could empathize with the retort of "OK tell them your opinion and let them decide")

( @vanillajonathan I hope it's OK us having this discussion here!)

**Comment by vanillajonathan** [comment]
Yeah, it is okay to have this discussion here, and there are pros and cons.

Pros are that it lowers the barrier to entry for application developers. But one could argue that app icons should be unique anyway so it is up to the developers to create their own app icons.

I believe it is fairly common practice for organizations to have a "press" or "press material" page on their website with links to logos and that is quite common for these to be offered in both a colored and a black/white version.
